### PR TITLE
Fixed reaction selection

### PR DESCRIPTION
**Pull request by ozerovs**

### CODE REVIEW

**Review by eliasbraun — APPROVED** [review]
Only one little comment

File: packages/components/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "bugs": {
     "url": "https://github.com/goodhood-eu/goodhood/issues"
   },
-  "version": "7.5.0",
+  "version": "7.6.0",
```
[the remark this answers is not in the document]
Comment (ozerovs) [reply packages/components/package.json:14]: I was thinking of it, but somehow felt like 7.6.0 suits more with the gut feeling. Would appreciate other thoughts.

[the remark this answers is not in the document]
Comment (peterhass) [reply packages/components/package.json:14]: --> bugfix

no new feature added, just fixed an old one -> bugfix

File: packages/components/src/reaction_selection_menu/hooks.jsx [hunk 2]
```diff
@@ -128,7 +128,7 @@ export const useHoverIndexCalculator = (itemRefs) => {
 
     const positionOnScreen = screenPosition(items[0]);
     const positions = items.map((item) => {
-      if (!positionCache.has(item)) positionCache.set(item, position(item).left);
+      if (!positionCache.has(item)) positionCache.set(item, screenPosition(item).left);
```
Comment (peterhass) [inline packages/components/src/reaction_selection_menu/hooks.jsx:131]: lol

**Review by peterhass — APPROVED** [review]

File: packages/components/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (rejas) [reply packages/components/package.json:14]: --> bugfix

**Review by rejas — APPROVED** [review]

File: packages/components/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (ozerovs) [reply packages/components/package.json:14]: <-- bugfix

**Pull request merged**